EasyPost/Models/API/Beta/StatelessRate.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using EasyPost._base;
+using Newtonsoft.Json;
+
+namespace EasyPost.Models.API.Beta
+{
+    public class StatelessRate : EphemeralEasyPostObject
+    {
+        #region JSON Properties
+
+        [JsonProperty("billing_type")]
+        public string? BillingType { get; set; }
+        [JsonProperty("carbon_offset")]
+        public CarbonOffset? CarbonOffset { get; set; }
+        [JsonProperty("carrier")]
+        public string? Carrier { get; set; }
+        [JsonProperty("carrier_account_id")]
+        public string? CarrierAccountId { get; set; }
+        [JsonProperty("currency")]
+        public string? Currency { get; set; }
+        [JsonProperty("delivery_date")]
+        public DateTime? DeliveryDate { get; set; }
+        [JsonProperty("delivery_date_guaranteed")]
+        public bool? DeliveryDateGuaranteed { get; set; }
+        [JsonProperty("delivery_days")]
+        public int? DeliveryDays { get; set; }
+        [JsonProperty("est_delivery_days")]
+        public int? EstDeliveryDays { get; set; }
+        [JsonProperty("list_currency")]
+        public string? ListCurrency { get; set; }
+        [JsonProperty("list_rate")]
+        public string? ListRate { get; set; }
+        [JsonProperty("rate")]
+        public string? Price { get; set; } // "Rate" is the enclosing class name
```

Comment:
`Rate` is the name of the class. In .NET, you can't have a property of a class that is the same name as the enclosing class.

Reply:
I realize now this is on StatelessRate, but to remain consistent with how it works in the normal `Rate` class (and since this will eventually become the default `Rate` class)